src/wasm/literal.cpp

```diff
@@ -223,9 +223,14 @@ void Literal::printDouble(std::ostream& o, double d) {
 
 void Literal::printVec128(std::ostream& o, const std::array<uint8_t, 16>& v) {
   o << std::hex;
-  for (auto i = 0; i < 16; ++i) {
-    o << "0x" << uint32_t(v[i]);
-    if (i < 15) o << " ";
+  for (auto i = 0; i < 16; i += 4) {
+    if (i) o << " ";
+    o << "0x" << std::setfill('0') << std::setw(8) << uint32_t(
```

Comment:
Sorry for the post-merge comment, but looking back at this, do we need to reset the ostream state after `std::setfill` and `std::setw`?

Reply:
I'm not overly familiar with this, but from what I've read `setw` affects the next output operation only. While `setfill` doesn't seem to be used anywhere else, saving and then resetting it might be good practice though.